### PR TITLE
Add basic samples to show Normals

### PR DESCRIPTION
**Pull request by torbsorb**

### CODE REVIEW

File: source/camera/advanced/capture_hdr_print_normals.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    print("Capturing frame (HDR)")
+    with camera.capture(settings) as frame:
+        point_cloud = frame.point_cloud()
+
+        print("Computing normals and copying them to CPU memory")
+        normals = point_cloud.copy_data("normals")
+
+        print("Printing normals for the central ")
+        radius_of_pixels_to_print = 5
+        print(f"{radius_of_pixels_to_print * 2} x {radius_of_pixels_to_print * 2} pixels")
```
Comment (SatjaSivcev) [inline source/camera/advanced/capture_hdr_print_normals.py:46]: it is a bit weird to have` radius_of_pixels_to_print = 5` between two prints. I would go for:

```
radius_of_pixels_to_print = 5
print(f"Printing normals for the central {radius_of_pixels_to_print * 2} x {radius_of_pixels_to_print * 2} pixels")
```

Comment (torbsorb) [reply source/camera/advanced/capture_hdr_print_normals.py:46]: done, but the formatter will split the print

File: source/camera/advanced/capture_hdr_print_normals.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    num_of_rows = normals.shape[0]
+    num_of_cols = normals.shape[1]
+    print(line_sep)
+    for row in range(int(num_of_rows / 2 - radius), int(num_of_rows / 2 + radius)):
+        for col in range(int(num_of_cols / 2 - radius), int(num_of_cols / 2 + radius)):
+            normal_string = f"Normals ({row}, {col}): ["
+            for i in range(normals.shape[2]):
+                normal_string += f"{normals[row, col, i]:.3} ".rjust(10)
+            normal_string += "]"
+            print(normal_string)
```
[the remark this answers is not in the document]
Comment (torbsorb) [reply source/camera/advanced/capture_hdr_print_normals.py:24]:
```
Connecting to camera
Configuring settings
Capturing frame (HDR)
Computing normals and copying them to CPU memory
Printing normals for the central 10 x 10 pixels
--------------------------------------------------
Normals (595, 955): [     x: -0.486     y: -0.0775      z: -0.871 ]
Normals (595, 956): [     x: -0.315     y: 0.00935      z: -0.949 ]
Normals (595, 957): [     x: -0.294       y: 0.148      z: -0.944 ]
Normals (595, 958): [     x: -0.382       y: 0.194      z: -0.903 ]
Normals (595, 959): [     x: -0.324        y: 0.29      z: -0.901 ]
Normals (595, 960): [     x: -0.328        y: 0.26      z: -0.908 ]
Normals (595, 961): [     x: -0.183       y: 0.339      z: -0.923 ]
Normals (595, 962): [      x: -0.22       y: 0.412      z: -0.884 ]
Normals (595, 963): [     x: -0.365       y: 0.347      z: -0.864 ]
Normals (595, 964): [     x: -0.523       y: 0.264       z: -0.81 ]
```

File: source/applications/basic/visualization/capture_hdr_vis_normals.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+        print("Visualizing normals in 2D")
+        display_rgb(rgb=rgba[:, :, :3], title="RGB image")
+        display_rgb(rgb=normals_colormap, title="Colormapped normals")
+
+        input("Press any key to continue...")
+
+        print("Visualizing normals in 3D")
+        display_pointcloud(xyz=xyz, rgb=rgba[:, :, :3])
+        display_pointcloud(xyz=xyz, rgb=normals_colormap * 255)
```
[the remark this answers is not in the document]
Comment (torbsorb) [reply source/applications/basic/visualization/capture_hdr_vis_normals.py:37]: 2D:
image

3D, our RGB:
image

normals as colors:
image

normals: (should downsample)
image

File: source/applications/basic/visualization/capture_hdr_vis_normals.py [hunk 4]
```diff
@@ -16,11 +16,12 @@ def _main():
     camera = app.connect_camera()
 
     print("Configuring settings")
-    settings = zivid.Settings(acquisitions=[zivid.Settings.Acquisition(aperture=fnum) for fnum in (11.31, 5.66, 2.83)])
+    settings = zivid.Settings(acquisitions=[zivid.Settings.Acquisition(aperture=fnum, brightness=1.0) for fnum in (11.31, 5.66, 2.83)])
 
     print("Capturing frame (HDR)")
     with camera.capture(settings) as frame:
         point_cloud = frame.point_cloud()
+        # point_cloud.downsample(zivid.PointCloud.Downsampling.by4x4)
```
Comment (SatjaSivcev) [inline source/applications/basic/visualization/capture_hdr_vis_normals.py:24]: why commented?

File: source/applications/basic/visualization/capture_hdr_vis_normals.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (SatjaSivcev) [reply source/applications/basic/visualization/capture_hdr_vis_normals.py]: I really like how normals as colors look like!

File: source/applications/basic/visualization/capture_hdr_vis_normals.py [hunk 5]
```diff
@@ -33,8 +34,9 @@ def _main():
         input("Press any key to continue...")
 
         print("Visualizing normals in 3D")
-        display_pointcloud(xyz=xyz, rgb=rgba[:, :, :3])
-        display_pointcloud(xyz=xyz, rgb=normals_colormap * 255)
+        print("n: for normals")
+        print("9: for point cloud colored by normals")
```
[the remark this answers is not in the document]
Comment (SatjaSivcev) [reply source/applications/basic/visualization/capture_hdr_vis_normals.py:38]: Ah, inputs.

**Pull request merged**